### PR TITLE
New login

### PR DESCRIPTION
**Pull request by fmassouNTTDATA**
maintenance of Login and script with Error

### CODE REVIEW

**Review by ZorbaDimatteo — APPROVED** [review]

File: src/test/java/it/pn/frontend/e2e/api/destinatario/SpidAcs.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                for (Header header : headers) {
+                    logger.info("Key : " + header.getName()
+                            + " ,Value : " + header.getValue());
+                }
+
+                Map<String, String> hm = new HashMap<>();
+                hm.put("responseCode", String.valueOf(classicHttpResponse.getCode()));
+                hm.put("urlPortale",classicHttpResponse.getHeader("Location").getValue());
+
+                //Assert.assertEquals(classicHttpResponse.getCode(),301, this.getSpidAcsEndPoint()+" risponde con : "+classicHttpResponse.getCode());
```
Comment (leleOFA) [inline src/test/java/it/pn/frontend/e2e/api/destinatario/SpidAcs.java]: delete comment

File: src/test/java/it/pn/frontend/e2e/api/destinatario/SpidDemoLogin.java [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    private final String organizationDisplayNameInput;
+    private final String samlRequestInput;
+    private final String relayStateInput;
+    private final String sigAlgInput;
+    private final String signatureInput;
+    private final String purposeInput;
+    private final String minAgeInput;
+    private final String maxAgeInput;
+    private final String retryInput;
+
```
Comment (leleOFA) [inline src/test/java/it/pn/frontend/e2e/api/destinatario/SpidDemoLogin.java:39]: miss comment to end input as in src/test/java/it/pn/frontend/e2e/api/destinatario/SpidDemoStart.java

File: src/test/java/it/pn/frontend/e2e/api/destinatario/SpidAcs.java (hunk 1 above)
Comment (fmassouNTTDATA) [reply src/test/java/it/pn/frontend/e2e/api/destinatario/SpidAcs.java]: done

File: src/test/java/it/pn/frontend/e2e/api/destinatario/SpidDemoLogin.java (hunk 2 above)
Comment (fmassouNTTDATA) [reply src/test/java/it/pn/frontend/e2e/api/destinatario/SpidDemoLogin.java:39]: done

**Pull request merged**